### PR TITLE
Nvictoria/coursera

### PR DESCRIPTION
**Pull request by vnvictoria**
Please review

### CODE REVIEW

**Review by Andrei5632 — APPROVED** [review]
LGTM

File: src/test/java/VictoriaTest.java [hunk 1]
```diff
@@ -7,7 +7,6 @@
 public class VictoriaTest extends BaseTest {
     @Test
     public void CourseraTest() {
-
```
Comment (anonymoustuna) [inline src/test/java/VictoriaTest.java]: You might put get getDriver() for website in here.

File: src/test/java/VictoriaTest.java [hunk 2]
```diff
@@ -7,7 +7,6 @@
...
         String SearchBoxXpath = "//div[@class='react-autosuggest__container']//input";
```
Comment (anonymoustuna) [inline src/test/java/VictoriaTest.java:10]: You might initialize it as a WebElement so you do not need the line 14 after that.

File: src/test/java/VictoriaTest.java [hunk 3]
```diff
@@ -7,7 +7,6 @@
...
         String search = "//button[2][@aria-label=\"Submit Search\"]//div[@class=\"magnifier-wrapper\"]";
```
Comment (anonymoustuna) [inline src/test/java/VictoriaTest.java:11]: You can initialize this line as a WebElement as well.

**Review by anonymoustuna — APPROVED** [review]

**Pull request merged**